skunk.gemspec

```diff
@@ -41,9 +41,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency "rubycritic", ">= 4.4", "< 5.0"
   spec.add_dependency "terminal-table", "~> 1.8.0"
 
-  spec.add_development_dependency "bundler", "~> 2.0"
   spec.add_development_dependency "byebug", "~> 11"
-  spec.add_development_dependency "codecov", "~> 0.1.16"
+  spec.add_development_dependency "codecov", "< 0.2"
```

Comment:
Why we need to change a dependency version for this feature?

Reply:
I was having issues with codecov but now that I'm thinking it was likely because of the Gemfile.lock. This change might not be relevant now so maybe reverting it makes sense. Do you have an opinion about this?

Reply:
Yeah maybe we can revert this change and the .lock file change too, since it changed because of this.

Reply:
I think we should remove the Gemfile.lock from the repo right?

Reply:
Yes, but I think that will be better in another PR